lib/kubernetes-deploy/deploy_task_config_validator.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class DeployTaskConfigValidator < TaskConfigValidator
+    PROTECTED_NAMESPACES = %w(
```

Comment:
Does this constant make sense with the changes to `krane deploy` to pass in the list of protected namespaces?

Reply:
No, we'll be able to remove that after the `krane deploy` changes are merged.